src/node_util.cc

```diff
@@ -145,28 +145,28 @@ static void GetHiddenValue(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
 
   CHECK(args[0]->IsObject());
-  CHECK(args[1]->IsUint32());
```

Comment:
I don't see the point of this change. It's a bug in `lib/internal/util.js` when `!args[1]->IsUint32()`.